object_intern.go

```diff
@@ -526,3 +518,66 @@ func (oi *ObjectIntern) Len(ptrs []uintptr) (retLn []int, all bool) {
 	}
 	return
 }
+
+func (oi *ObjectIntern) JoinStrings(nodes []uintptr) (string, error) {
```

Comment:
wouldn't it be nicer if there's a parameter for the character to join with? f.e. `.`, like in `strings.Join()`